src/common/TALENTS/priest.ts

```diff
@@ -360,6 +360,12 @@ const talents = createTalentList({
     icon: 'spell_shadow_chilltouch',
     maxRanks: 2,
   },
+  INESCAPABLE_TORMENT_TALENT_DAMAGE: {
```

Comment:
Any manual additions for talents need to go in the SPELLS file for the class, as the TALENTS file will be overwritten any time the talents are regenerated.

Reply:
That is great to know.  I moved the manual additions from TALENTS into SPELLS.